Formula/b/bigquery-emulator.rb

```diff
@@ -0,0 +1,35 @@
+class BigqueryEmulator < Formula
+  desc "Emulate a GCP BigQuery server on your local machine"
+  homepage "https://github.com/goccy/bigquery-emulator"
+  url "https://github.com/goccy/bigquery-emulator/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "4f8c037d03cd23d2a44d74460b5399213e0efeb33d6cade25bfce25499c4699a"
+  license "MIT"
+  head "https://github.com/goccy/bigquery-emulator.git", branch: "main"
+
+  depends_on "go" => :build
+
+  uses_from_macos "llvm" => :build
+  uses_from_macos "netcat" => :test
+
+  fails_with :gcc
```

Comment:
Does it fail with every GCC version? Do we know the reason why?

Reply:
idk why, I know it fails with gcc and maintainer [recommends using clang](https://github.com/goccy/bigquery-emulator?tab=readme-ov-file#install).